README.md

```diff
@@ -73,7 +74,9 @@ No modules.
 
 ## Outputs ##
 
-No outputs.
+| Name | Description |
+|------|-------------|
+| initial\_passwords | The initial password for each user, which must be changed at first login. |
```

Comment:
You should state that this output is a map, with keys equal to usernames and values equal to passwords.

Reply:
Please see https://github.com/cisagov/cool-users-non-admin/pull/51/commits/c965088cf38ffe06c1706e0e45d9803528e6e3fc.